### ISSUE TITLE
关于reg_u及超参

### ISSUE THREAD

**Issue by JayW-er** [MiracleHH/RecommPoison#1]
请问代码中的reg_u的作用，以及paper中的η与λ超参数在代码中分别用哪个变量来表示。loss = alpha * reg_u * loss以及loss = alpha * (label_poisoned ** 2).sum()中的alpha、reg_u是否就是paper中的超参数η与λ？

**Comment by MiracleHH** [comment]
> 请问代码中的reg_u的作用，以及paper中的η与λ超参数在代码中分别用哪个变量来表示。loss = alpha * reg_u * loss以及loss = alpha * (label_poisoned ** 2).sum()中的alpha、reg_u是否就是paper中的超参数η与λ？

@topWangZY Thanks for your interest in our work. `reg_u` and `alpha` in the code represent `η` and `λ` in the paper respectively. `η` is a coefficient to balance the attack goal and the number of ratings of one fake user.

**Comment by JayW-er** [comment]
非常感谢您百忙中的解答，感谢！



---- Replied mail ----
| From | Hai ***@***.***> |
| Date | 08/10/2022 23:45 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [MiracleHH/RecommPoison] 关于reg_u及超参 (Issue #1) |

Closed #1 as completed.
